stable/external-dns/templates/deployment.yaml

```diff
@@ -207,6 +210,21 @@ spec:
           value: {{ .Values.aws.credentials.mountPath }}/credentials
         {{- end }}
         {{- end }}
+        # Azure Private DNS variables 
+        {{- if eq .Values.provider "azure-private-dns" }}
+        {{- if .Values.azure.tenantId }}
+        - name: AZURE_TENANT_ID
+          value: {{ .Values.azure.tenantId }}
+        {{- end }}
+        {{- if .Values.azure.aadClientId }}
+        - name: AZURE_CLIENT_ID
+          value: {{ .Values.azure.aadClientId }}
+        {{- end }}
+        {{- if .Values.azure.aadClientSecret }}
+        - name: AZURE_CLIENT_SECRET
+          value: {{ .Values.azure.aadClientSecret }}
+        {{- end }}
+        {{- end }}
```

Comment:
The new chart parameters should be documented in the README (https://github.com/helm/charts/blob/master/stable/external-dns/README.md#parameters)

Reply:
@sameersbn The readme already has these parameters. It wasn't used in the templates though. 
It's under:
* azure.resourceGroup
* azure.tenantId
* azure.aadClientId
* azure.aadClientSecret
* azure.subscriptionId

Reply:
Hi there. I also had a quick look and effectively the PR is only setting the missing environment variable and not introducing any new parameters. Please review asap because external-dns is currently totally broken for Azure private DNS.

Reply:
@sameersbn any updates?